### DISCUSSION TITLE
Radar (SAM) Facility compare to multiple e.g. Tall Rack

### DISCUSSION THREAD

**Discussion by Parel803** [PygmalionOfCyprus/cmo-db-requests#3137] [Q&A]
Good afternoon,

Just a question on how to ... in facility. I ran into the Tall Rack radar.
The facility Tall Rack for the 55Zh6M Nebo M consists of three seperate facilities (1846, 1847 & 1848) with each there own radar, sensor: 4556, 4557 & 4558). 
Not important but the command post is than missing.
How do we see in a case like this the facility, in this case four trucks involved (that is without a SAM).

If I think, needs more searching the internet, one of the sensor Tall Rack isn't a Tall Rack (55K6-3). Can I drop that in the issues, realizing that deleting of stuff is hard due to scenario's made. Sort of similar might be in the DB concerning versions.

Both not important but sometime like to do my pre-retirement work :-)   I'm just curious in the standards in some area's so it is easier to add DB options, images and descriptions.

best regards GJ

**Comment by PygmalionOfCyprus** [comment]
If you see any discrepancies feel free to put in a ticket, I'll do what I can.

**Reply by Parel803** [reply]
I will, thx for your answer.
